scripts/fetch/precip_grid_data.R

```diff
@@ -0,0 +1,23 @@
+
+
+fetch.precip_grid_data <- function(viz){
+  deps <- readDepends(viz)
+  checkRequired(deps, c("grid"))
+  grid <- deps[['grid']]
+  sp_grid <- as(grid, "Spatial")
```

Comment:
in my experience with the gdp viz, using `as( obj, "Spatial")` required the `sf` package to be loaded within the function because you need the sf method for `as`.

Reply:
yes, I think this works a little differently or easier in this case because `remake` is in play instead of `make`.

Reply:
oh nice!